src/main/java/com/nike/cerberus/server/config/guice/CmsMyBatisModule.java

```diff
@@ -25,14 +27,16 @@
  */
 public class CmsMyBatisModule extends MyBatisModule {
 
+    private final Config config = ConfigService.getInstance().getAppConfigMergedWithCliGeneratedProperties();
+
     @Override
     protected void initialize() {
         bindDataSourceProviderType(C3p0DataSourceProvider.class);
         bindTransactionFactoryType(JdbcTransactionFactory.class);
 
         addSimpleAliases("com.nike.cerberus.record");
         addMapperClasses("com.nike.cerberus.mapper");
-        useCacheEnabled(false);
+        useCacheEnabled(config.getBoolean("cms.mybatis.cache.enabled"));
```

Comment:
In what use cases will this be set to true? I see it is set to false by default in `src/main/resources/cms.conf` below.

Reply:
Those are the default props, they can be overridden in environment specific props files as well as with the Cerberus LC CLI, which is how we set things like this in addition to API keys and the like.

Reply:
```bash
AP77JGH6A84FA3:~ jfiel2$ c_lm -e dev update-cms-config -Pcms.mybatis.cache.global.repeatReadCounterResetInSeconds=2 -Pcms.mybatis.cache.global.repeatReadThreshold=2


AP77JGH6A84FA3:~ jfiel2$ c_lm -e dev view-config --config-path cms/environment.properties | grep cache
cms.auth.iam.kms.cache.enabled=true
cms.encryption.cache.decrypt.maxAgeInSeconds=30
cms.encryption.cache.decrypt.maxSize=1000
cms.encryption.cache.enabled=true
cms.encryption.cache.encrypt.maxAgeInSeconds=30
cms.encryption.cache.encrypt.maxSize=1000
cms.encryption.cache.encrypt.messageUseLimit=5
cms.mybatis.cache.enabled=true
cms.mybatis.cache.global.repeatReadCounterResetInSeconds=2
cms.mybatis.cache.global.repeatReadThreshold=2
AP77JGH6A84FA3:~ jfiel2$ 
```